quaternion_attitude_control/README.md

```diff
@@ -1,4 +1,14 @@
-# Multicopter control allocation
+# Quaternion attitude control
 The algorithm implements an attitude control strategy for flying vehicles especially multicopters using unit quaternion rotation math.
 
+To run the script you need Python 3 (tested on 3.7.1) and the following python modules:
+```
+python -m pip install numpy matplotlib pyquaternion
+```
+
+Command to run the script:
+```
+python quaternion_attitude_control_test.py
```

Comment:
my computer run need this： `python3 quaternion_attitude_control_test.py`

Reply:
ok, i switched to `python3` and `sudo` for pip to cover all necessities.